templates/reservations.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h1>Your Reservations</h1>

```

Comment:
It will be good if we can somehow show to the user two sections, one for past bookings and one for the upcoming bookings.

Reply:
You can do this in the next PR, not necessary to do in this one.

Reply:
ok didi